MidasCivil_Adapter/AdapterActions/Execute.cs

```diff
@@ -319,6 +319,29 @@ public async Task<bool> RunCommand(Analyse command)
         }
 
         /***************************************************/
+        public async Task<bool> RunCommand(Import command)
+        {
+            if (m_midasCivilVersion == "9.5.0.nx")
+            {
+                string filePath = command.FilePath;
+                if (File.Exists(filePath))
+                    filePath = filePath.Replace("\\", "\\\\");
+                else
+                    Engine.Base.Compute.RecordError("The given file path does not exist.");
```

Comment:
I think in here we should have some further checks - you can check the file extension and return an error if it's not .mct or .txt.

There is another [Import](https://support.midasuser.com/hc/en-us/articles/35994338816793-Import-to-Json) method that we should also add support for in this PR.